service/src/main/kotlin/fi/espoo/evaka/invoicing/data/FeeDecisionQueries.kt

```diff
@@ -509,6 +510,31 @@ fun findFeeDecisionsForHeadOfFamily(
         .let { it.merge() }
 }
 
+fun approveRetroactiveFeeDecisionDraftsForSending(h: Handle, ids: List<UUID>, approvedBy: UUID) {
+    val sql =
+        """
+        UPDATE fee_decision
+        SET
+            status = :status,
+            decision_number = nextval('fee_decision_number_sequence'),
+            approved_by = :approvedBy,
+            decision_handler = :approvedBy,
+            approved_at = :now
+        WHERE id = :id
+        """.trimIndent()
+
+    val batch = h.prepareBatch(sql)
+    ids.map { id ->
+        batch
+            .bind("id", id)
+            .bind("status", FeeDecisionStatus.WAITING_FOR_SENDING.toString())
+            .bind("approvedBy", approvedBy)
+            .bind("now", LocalDateTime.now())
+            .add()
+    }
+    batch.execute()
+}
+
```

Comment:
Instead of creating a whole separate function for this I would have added a `useUnitHandlerOrSomething` parameter to `approveFeeDecisionDraftsForSending` and moved the
```
CASE
    WHEN youngest_child.finance_decision_handler_id IS NOT NULL AND fd.decision_type = 'NORMAL'
        THEN youngest_child.finance_decision_handler_id
    ELSE :approvedBy
END
```
part to kotlin code as well.